### DISCUSSION TITLE
Investigating how overcharging works

### DISCUSSION THREAD

**Discussion by MandrakeHorse** [CorsixTH/CorsixTH#2428] [Research Room]
This discussion concerns research into the mechanics of [overcharging patients](https://github.com/CorsixTH/CorsixTH/wiki/Research#over-charging), which is performed by raising treatment charges as a percentage via the [Drug Casebook](https://github.com/CorsixTH/CorsixTH/wiki/Game-Guide#drug-casebook).

In Theme Hospital, patients are very sensitive to even small price increases; if they decide a treatment costs too much, they will leave the hospital without paying and uncured, lowering your reputation and discouraging patients from coming to your hospital. CorsixTH currently (0.67) does not implement this behaviour and since patients will always cough up the cash, there is little reason not to set all your prices to maximum (200%) and watch the money roll in, potentially breaking the game.

To perform this experiment (using the GOG version via DOSBox), I started a new game on Level 1 (Toxicity) with Doctor (Normal) difficulty and quickly built a basic hospital with a GP's office, General Diagnosis, Psychologist, Pharmacy, Ward, Inflation, and Staff Room, with enough staff to keep these rooms occupied. Once I was ready, I opened my Hospital on 10 Jan and created a save file (which is attached below). Then I simply let the game run with minimal intervention until 1 Oct (i.e. upon reaching the level complete screen) while manually counting the number of patients who refused to pay, after which I counted the number of visits per room. I then reloaded my save file, increased all my prices (including those for new treatments as soon as they were discovered), and reran the simulation.

  | 100% | 101% | 102% | 103% | 104% | 105%
-- | -- | -- | -- | -- | -- | --
GP's office | 31 | 16 | 20 | 24 | 27 | 15
General Diagnosis | 7 | 5 | 2 | 3 | 4 | 2
Psychologist (diagnosis) | 0 | 1 | 4 | 6 | 4 | 2
Psychologist (cure) | 3 | 0 | 2 | 2 | 0 | 1
Pharmacy | 6 | 4 | 3 | 4 | 8 | 4
Ward | 7 | 0 | 4 | 2 | 5 | 3
Inflation | 3 | 2 | 2 | 2 | 1 | 1
**Total visits** | **57** | **28** | **37** | **43** | **49** | **28**
Cured | 12 | 6 | 7 | 8 | 9 | 6
**Didn't pay** | **0** | **0** | **3** | **6** | **5** | **5**

As you can see, there's an awful lot of randomness in play, such as the chance for a patient to spawn, what type of disease they have (such as Invisible patients, who can't be fully diagnosed and will eventually leave), and which diagnosis room the patient will be referred to. Even so, we can see that every patient paid for treatment when the prices were set to 100-101% (for some reason, the total number of visits was notably lower for 101%). From 102% onwards, however, we start to see a small number of patients leaving without paying, having a knock-on effect on reputation and potentially reducing patient intake (note that 100% was the only run where the number of cured patients exceeded the level's goal of 10).

To get a clearer picture of how significantly overcharging affects patient behaviour, I repeated the experiment with larger price increments:

  | 110% | 115% | 120% | 125% | 150% | 200%
-- | -- | -- | -- | -- | -- | --
GP's office | 17 | 13 | 9 | 9 | 2 | 0
General Diagnosis | 3 | 1 | 2 | 1 | 0 | 0
Psychologist (diagnosis) | 1 | 0 | 1 | 0 | 0 | 0
Psychologist (cure) | 1 | 0 | 0 | 0 | 0 | 0
Pharmacy | 2 | 2 | 1 | 0 | 0 | 0
Ward | 2 | 4 | 0 | 1 | 0 | 0
Inflation | 1 | 0 | 0 | 0 | 0 | 0
**Total visits** | **27** | **20** | **13** | **11** | **2** | **0**
Cured | 4 | 2 | 1 | 0 | 0 | 0
**Didn't pay** | **6** | **16** | **18** | **21** | **17** | **18**

Observations
- At some point between 120% and 125%, all patients will begin leaving without paying, either at the reception desk or otherwise before they can be cured.
- Patients decide whether they want to pay for treatment before they start walking towards the room. If they decide to pay, the game remembers what the treatment's price was at the time of their decision, so you can't slow time and jack up prices to 200% before they arrive - they will pay whatever the price was when they were referred.
- When prices are moderately high (115-125%), some patients will pay for the GP Office and then leave without paying for further diagnosis. Others will pay for both the GP Office and further diagnosis, but leave without revisiting the GP Office. Interestingly, when prices were set to 150%, exactly two patients were willing to pay for GP Office (although not for further diagnosis). This all suggests that the chance to refuse payment is randomly rolled before they head towards the room, and that the higher the price, the more likely they'll refuse to pay.

Save file: [overpricingSaveFile.zip](https://github.com/CorsixTH/CorsixTH/files/12775565/overpricingSaveFile.zip)
Notable events:
- Invisible patients are ignored until they go home
- The Handyman always demands a pay rise in July, and I would always accept his demand to keep him working
- Most of the staff goes to the Staff Room from August until September
- VIP will visit automatically in August and gives moderate or positive response

**Comment by ageekhere** [comment]
I wonder if it is even worth the risk of losing a patient by raising the price?

**Reply by MandrakeHorse** [reply]
Once you've got your hospital up-and-running, it might be tempting to slightly overcharge to make a slightly more money in the short-term at the cost of your reputation. The problem is that if a patient decides to leave, they won't pay for the cure, which is where the majority of your income should come from. For example, a patient with Kidney Beans walking out means that you'll lose out on a healthy ~$1080 plus any further diagnosis and GP appointments, which ultimately cancels out the extra money you were squeezing from him through the latter (which will only be $1-2 per percent). Patients walking out also hits your reputation, which discourages patients from going to your hospital, thus reducing your overall income.

There might be a sweet spot where the chance of patients walking out is low enough that it's worth the risk, but it'll probably be somewhere very low (<105%?). You'd also get fewer walkouts if you keep your GP/diagnosis rooms at 100% but overcharge on cures (which earns more per percent, but infrequently so).

(Sidenote: My experiment only investigated the first ten months of operation to minimise variation and increased prices universally, meaning that most patients who walked out never even saw the GP. Future experiments could be performed with successful late-game hospitals, or only increasing the prices of cures instead of GP/diagnosis.)

**Reply by MandrakeHorse** [reply]
Another factor is that each treatment's base price is dependant on your Reputation (as well as the treatment's success rate). The higher your Reputation, the more patients will visit your hospital _and_ the more you'll automatically charge per visit. Therefore if you focus on curing as many patients as possible and minimise quitters, you'll end up making more money in the long run compared to manually increasing prices and potentially angering your patients into leaving (hence why I was only able to complete Level 1 by October when prices were left at 100%)

**Comment by MandrakeHorse** [comment]
Out of curiosity, I decided to repeat my experiment but with undercharging patients (performed by lowering treatment charges via the Drug Casebook). According to Theme Hospital's manual, "undercharging increases the number of admissions to the Hospital and potentially raises your Reputation, but each treatment earns you less." Once again, I set the prices for all treatments to the same percentage and then reran my hospital with minimal intervention between 10 Jan and 1 Oct, after which I counted the number of visits per room and total number of patients cured.

  | 100% | 95% | 90% | 85% | 80% | 75% | 50%
-- | -- | -- | -- | -- | -- | -- | --
GP's office | 31 | 29 | 32 | 25 | 18 | 27 | 24
General Diagnosis | 7 | 5 | 6 | 6 | 1 | 7 | 3
Psychiatric | 3 | 6 | 8 | 4 | 5 | 3 | 5
Pharmacy | 6 | 9 | 3 | 4 | 5 | 7 | 8
Ward | 7 | 4 | 6 | 4 | 5 | 4 | 3
Inflation | 3 | 2 | 4 | 4 | 0 | 2 | 2
Total visits | 57 | 55 | 59 | 47 | 34 | 50 | 45
Cured | 12 | 12 | 13 | 11 | 7 | 10 | 10

What I discovered was that in contrast with overcharging, undercharging does not appear to have any impact on the number of admissions. In fact, there was one case (85%) where the total number of cured patients was actually _lower_ than average (possibly due to more untreatable Invisibility patients being randomly spawned and reducing the hospital's reputation). This does make some logical sense: patients don't know what's wrong with them before they're admitted, so you shouldn't be able to attract more patients with, say, Bloaty Head to your hospital by slashing the Inflator's price. However, considering that it has been proven that patients will always pay for treatment at 100% price, there seems to be little reason to ever reduce prices below the default value, aside from a self-imposed challenge.

Future improvements to this experiment:
* Run the simulation for longer period to average out the randomness and investigate long-term effects
* Set policy so staff only rest at 100% tireness, thus delaying the stage when everyone heads to the staff room all at once

**Comment by MandrakeHorse** [comment]
I made these graphs to better visualise my test data. There appears to be an exponential relationship between overcharging and total visits, while there is no clear relationship between undercharging and total visits.
![image](https://github.com/CorsixTH/CorsixTH/assets/142453312/94ff40fa-1982-4ba7-a60a-43169608d991)
![image](https://github.com/CorsixTH/CorsixTH/assets/142453312/11cbac80-1ec0-427d-8b07-16c6e5568f0a)

**Reply by lewri** [reply]
Out of interest, I wonder if the hospital reputation was also changing over the test period.

**Reply by MandrakeHorse** [reply]
Reputation drops significantly whenever a patient refuses to pay and leaves (from 500, to 476, to 454, to 434, to 416, to 400 for the first five patients), which means that overcharging ultimately reduces the number of arrivals. Otherwise, Reputation would have gradually increased over time with every cured patient, before receiving a large boost from the VIP's visit in August - there was always an increase in arrivals afterwards, although this wouldn't have affected the data since everyone went to the staff room en masse between September to October, blocking any further visits to the hospital's rooms.

**Comment by mugmuggy** [comment]
You can take a look at #1986, function World:chooseHospitalForPatient(disease). To get some more idea of how TH chooses spawns and the L1 file:
#gbv.AllocRand			4	Population Allocation - chance of random hospital selection
#gbv.AllocTotalRep		1	Population Allocation - total reputation weighting
#gbv.AllocIndRep			2	Population Allocation - illness reputation weighting
#gbv.AllocDelay			3	Population Allocation - months delay before proper allocation

It waits 3 months, form memory April (AllocDelay) will start the proper allocation, before that it is just round robin to the hospitals. Then after that there is a 'random' allocation done in 1 in 4 allocations (AllocRand). Then the hospitals are weighted and checked. If the hospital hasn't discovered the disease the hospital reputation is used instead. If the prices are relatively too cheap, the 50% (minimum) , would imply a 250 rep 'just to get a visitor'. They do evaluate if its too cheap as well. As your reputation for undiscovered is hospital rep, this influences it as well.

And to top it off the spawn rate (across all hospitals) is consistent per month with the level file also and the computer players also have different thresholds for curing, and that leads to one generally performing badly, one performing okish and one being competitive.

I'm pretty sure that random allocation plays a big part in what you are seeing, ie '25%' of the time, they don't look at the price. I say 25% but PRNG can be skewed.

**Comment by ARGAMX** [comment]
Nice job! 👍
Here is Level 12 Battenberg save from original TH with my automized hospital. 
Only machines periodically require replacement.
May be it will be useful for any experiments like this.

[artem.h1.zip](https://github.com/user-attachments/files/17174820/artem.h1.zip)

**Comment by ARGAMX** [comment]
As MandrakeHorse previously noted in this topic:
> CorsixTH currently (0.67) does not implement this behaviour and since patients will always cough up the cash, there is little reason not to set all your prices to maximum (200%) and watch the money roll in, potentially breaking the game.

So, currently, the functionality for patient refusing payment in CorsixTH is not aligned with the original game.

This is fairly easy to spot if you raise all prices to 120% in Casebook. 
In the original TH, at 120%, you'll see a significant number of patient refusals at this point. 
In CorsixTH, at 120%, you'll barely notice the difference.
You'll also notice discrepancies on other increased cost values.
At 150%, your hospital in the original TH can no longer function because there simply won't be anyone to treat.
In CorsixTH, at 150%, you'll see patients refusing, but your hospital will generally continue to function.

So, the difference is significant.
Many players notice this and take advantage of it.
And it looks like legal cheating.

I decided to tackle this problem first and conducted a search to see what we have at the moment.
The first thing I found is this MandrakeHorse's research into how this functionality works in the original TH.

Before implementing or fixing something in CorsixTH, I usually get the idea to research how it works in the original game.
It seemed obvious to me and I thought that everyone would do the same.
From this perspective, MandrakeHorse's research was a logical and reasonable starting point on this issue.

From his research we can notice: 
1. In the original Theme Hospital, patients are very sensitive to even small price increases; Already at 102% we start to receive patients' rejections.
2. Already at 125%, the number of patients rejections is simply colossal.
3. Rejection rate is subject to some degree of randomness.
4. At 100% price levels, no one refuses to pay.

--------------

I decided to repeat the MandrakeHorse's experiment, but not on the first level, but on the last one, on the 12th level - Battenberg. 
Because:
1. I wanted to reduce the randomness factor by increasing the patient sample by 4 times.
2. I also wanted to exclude the possibility of patients leaving for other reasons - unable to diagnose, fed up etc.
3. I also wanted to make separate experiments for diagnostic rooms and treatment rooms.
4. I also wanted patients to have only one chance to reject. That is, if we make a patient make a decision about paying an inflated price 4 times — 3 times in diagnostic rooms and 1 in the treatment room — the chance of him leaving is obviously higher than if we only ask him only once. Leaving at any stage interrupts their treatment. So it is necessary to conduct a separate experiment only with treatment rooms. In the MandrakeHorse's research, you can see a large drop between 100% and 105% on the graph. And then, between 105% and 110% and later, there was no longer any such big drop. It was precisely this drop that I wanted to eliminate from my results by this point.

You can see that two years ago in this thread, I suggested a savegame that could be used to conduct just such an experiment. Well, that means it's time to do it myself.

I used the same save suggested in the thread.
1. I loaded that save.
2. Then deleted the reception desks and waited several months. This way, I cleared the hospital of patients.

3. Then I made a new save at the beginning of the year in January and it was from this place that I started all the experiments.

4. First, I bought two reception desks, installed them, and set the speed to 5.
5. On December 1st, I clicked on the reception desk and checked how many patients had passed through them — that's the number of patients who arrived at the hospital during the experiment.
6. Then I removed both reception desks to stop new patients from arriving and waited another 3 months for the hospital patients to leave.
7. After these 3 months I check the count of patients is cured on the Progress Report screen.
8. I subtract from it the amount that was before the start of the experiment.

Next, I repeated starting from point 4, setting different percentages for treatment rooms.

This gives me with three numbers:

1. Current price percentage. (100% - 200%)
2. Number of patients arrived. (X)
3. Number of patients treated. (Y)

I calculated the number of patients who refused (Z) as follows: Z = X - Y

While running an experiment, I noticed that in the original TH, the cursor position in Casebook is saved between entries. In CorsixTH 0.69, it is not.
Also you can use buttons ← and → on your keyboard to change price value. But the price only changes by 1%, so it’s not exactly a very effective method.
Also if you press P to pause you can't open Casebook. But if you open Casebook first, you can press P and your game will pause, although visually it will not be visible.

First, I ran a base experiment with 100% prices.
Then I repeated the experiment with values:  101%, 105%, 110%, 115%, 120%, 125%, 150%, 175%, 195%, 200%

I didn't change the diagnostic prices, as described above. They always remained at 100%, covering treatment only. So each patient only had 1 chance to make a reject.

I got the following results:

X | 100% | 101% | 105% | 110% | 115% | 120% | 125% | 150% | 175% | 195% | 200%
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | 
Reception Visitors | 231 | 241 | 226 | 221 | 206 | 220 | 211 | 190 | 166 | 142 | 131
Cured | 231 | 236 | 184 | 162 | 112 | 106 | 72 | 30 | 5 | 1 | 0
Didn't pay | 0 | 5 | 42 | 59 | 94 | 114 | 139 | 160 | 161 | 141 | 131
Percentage of those who paid | 100% | 97,92% | 81,41% | 73,30% | 54,36% | 48,18% | 34,12% | 15,78% | 3,01% | 0,7% | 0%

Let's plot the obtained results on the XY coordinate plane:

The OX axis will show the overpayment percentage.
The OY axis will show the percentage of patients who agreed to pay.

Thanks [www.agentsfordata.com/graphs/x-y](http://www.agentsfordata.com/graphs/x-y) for this visualisation.

<img width="1456" height="736" alt="scatter-plot_1" src="https://github.com/user-attachments/assets/012d0ba7-9293-4f5c-bad0-3dc70856e18a" />

Let's try to make function which would go through the obtained points.
I had to remove several points from the graph. But the essence remains the same.

<img width="1408" height="894" alt="scatter-plot_2" src="https://github.com/user-attachments/assets/67e2c08a-4598-48ae-b03a-6b6b2ee5310a" />

We see that the results are nonlinear and hyperbolic in nature.

I went through various functions that could describe the graph passing near the obtained points.

Thanks [www.desmos.com/calculator](http://www.desmos.com/calculator) for this visualisation.

<img width="1272" height="493" alt="537166266-a501652a-a856-41d5-aabb-ad8ab22d1ce6_2" src="https://github.com/user-attachments/assets/8405312a-90cb-47a8-b6a1-d045f6567fe8" />


I like the simpler functions with as few constants as possible the most.
It's unlikely that Bullfrog used some complicated function with lots of constants.

Among these, I liked one power function and one exponential function the best.
They gave a pretty good approximation to the experiment result data.

<img width="1481" height="734" alt="537166526-1e79b04a-c41d-40f3-9351-044dd0690756_2" src="https://github.com/user-attachments/assets/c8a1739a-ad2e-4dd0-9679-d10777c8f5f6" />


Both functions was pretty good and it was possible to stick with either one.
I was choosing between them, but decided to go with the exponential one function with -0.04 const.
I don't know why.

--------------

I was raising prices only for treatment rooms. So each patient was able to “roll the dice” once.
But because of this, I did not cover the diagnostic room by experiment.
Let's fix this.

Visit to a diagnostic room costs the patient much less than a treatment room. 
I was interested in testing the hypothesis of whether room price impacts refusal probability.
Therefore, among the diagnostic rooms, I chose the cheapest one – the GP office.

I restarted the experiment and set the GP price for 175%. 
This raised the GP price from $200 to $350.
And launched the simulation.
In summary 252 patients passed through the reception.
Among 252 patients 247 refused to pay for their first GP visit.
So only 5 agreed. Among these 5 only 2 were cured, because they were diagnosed by the first visit.
The remaining 3 could not afford to pay for a second visit to the GP office.
This result gives us ‘Percentage of those who paid’ at 2%.
In the experiment with the treatment room at 175%, we got 3% of patients who agreed to pay, which is even more.

I found 2% and 3% to be close enough to assume overprice refusal works with diagnostic rooms just the same as with treatment rooms. So it is not necessary to conduct an experiment with diagnostic rooms at other values.

From this, we can conclude that patients don't care whether they overpay $10 at the GP's office or $100 at the treatment room. Patients will agree to pay $2,000-$3,000 in treatment rooms if the price is set at 100%, but they can't afford to overpay $150 at the GP's office on 175%.

In other words, the absolute amount is irrelevant to patients.

Okay. I think we can wrap up the experiments here. Everything is pretty clear now.

--------------

Now that I thought I knew what to do, I decided to look at the code…

Functionality responsible for patients payment refusals was implemented in 2014-2015, but it's very far from the original game, and it's unclear why it was made that way.

I found that the following functions were responsible for this functionality:

`function Patient:agreesToPay(disease_id)`
and
`function Patient:getPriceDistortion(casebook)`

`hospital.lua`
```Lua
  -- Price distortion level over which the patients might consider the
  -- treatment to be over-priced.
  local over_priced_thresholds = {0.4, 0.3, 0.2}
  self.over_priced_threshold = over_priced_thresholds[difficulty]
```

`patient.lua`
```Lua
--! Returns true if patient agrees to pay for the given treatment.
--!param disease_id (string): The id of the disease to test
function Patient:agreesToPay(disease_id)
  local hosp = self.hospital
  local casebook = hosp.disease_casebook[disease_id]
  local price_distortion = self:getPriceDistortion(casebook)
  local is_over_priced = price_distortion > hosp.over_priced_threshold

  if is_over_priced and math.random(1, 5) == 1 then return false end
  self.pay_amount = hosp:getTreatmentPrice(disease_id)

  return true
end
```

```Lua
--! Estimate the subjective perceived distortion between the price level the
--! patient might expect considering the reputation and the cure effectiveness
--! of a given treatment and the staff internal state.
--!param casebook (table): casebook entry for the treatment.
--!return (float) [-1, 1]. The smaller the value is, the more the patient
--! considers the bill to be under-priced. The bigger the value is, the more
--! the patient patient considers the bill to be over-priced.
function Patient:getPriceDistortion(casebook)
  -- weights
  local happiness_weight = 0.1
  local reputation_weight = 0.6
  local effectiveness_weight = 0.3

  -- map the different variables to [0-1] and merge them
  local reputation = casebook.reputation or self.hospital.reputation
  local effectiveness = casebook.cure_effectiveness

  local weighted_happiness = happiness_weight * self:getAttribute("happiness")
  local weighted_reputation = reputation_weight * (reputation / 1000)
  local weighted_effectiveness = effectiveness_weight * (effectiveness / 100)

  local expected_price_level = weighted_happiness + weighted_reputation + weighted_effectiveness

  -- map to [0-1]
  local price_level = ((casebook.price - 0.5) / 3) * 2

  return price_level - expected_price_level
end
```

What do we see here?

It turns out that at values ​​of 101%, 105% and 110%, patients might not even consider the price to be higher than normal!
This would give `is_over_priced = false` and `Patient:agreesToPay` just always returns `true`.
And in this case, the random function is not even called.

As we can see, this does not follow the original TH logic at all, where patients consider the price to be overpriced starting from 101% and start to use a random function.

In CTH 0.69 patients start to consider the price overpriced starting from 120% or 130% or 140%, depending on the chosen difficulty of the game. Interesting, but not relevant to the original TH.

But that's not all...
`if is_over_priced and math.random(1, 5) == 1 then return false end`

So, if the patients consider the price overpriced, they refuse to pay only in 20% of cases!

So it makes no difference at all whether you set the price at 140% or 200%! The probability of rejection is the same! This also doesn't correspond to the original TH, where the probability of rejection always depends on the chosen overpayment percentage.

It's curious, but that logic can be checked in 10 minutes for inconsistency with the original game.
You don't even need to conduct research with exact values like mine or MandrakeHorse's to do that. Just play around with different percentages in already built hospitals in the original game and just look how many patients leave. Of course, this is still better than nothing.

But this creates a false illusion among players that this functionality is implemented and working correctly. And perhaps among the developers. Although, in reality, it shouldn't be used at all, as it provides a very significant advantage.
Oh well.

The most difficult thing I encountered at this stage was the function
`function Patient:getPriceDistortion(casebook)`

As you can see, it uses patient happiness, hospital reputation, and treatment effectiveness. 
However, it uses these factors not for the random function, but for the `is_over_priced` flag calculation.
This flag determines whether the patient will consider the price to be overpriced.

Such functionality, even if it was in the original game, would be quite difficult to check for consistency with the original game, and would take more than 10 minutes.

I can't say for sure that the original game doesn't take patient happiness into account. But personally, I don't think it does.

I also couldn't figure out how to elegantly integrate this code into the new logic and new formula.

So my suggestion is not to use happiness, efficiency, and reputation when trying to calculate the probability of a patient's refusal.

I also checked the original game's `.sam` files, which are analogous to our `.level` files, and didn't find any values ​​related to calculating the probability of rejection. So, I assumed that game difficulty doesn't affect the rejection probability. But you can recheck this. I haven't conducted an experiment to test this.

I also analysed the topics #266, #313, #971, #993. 
This is where these changes were made and discussed. 
And I still don’t understand why the refusal to pay was implemented in this way. 
So I think that someone simply came up with that logic without checking how the original game works. 
This is strange to me. So I have no motivation to keep old logic and use it in the algorithm I propose.

So I think instead of this code
```Lua
if is_over_priced and math.random(1, 5) == 1 then return false end
```
should be a code like this
```Lua
  local overprice_size = casebook.price - 1.0
  local payment_chance_modificator = 4
  local payment_chance = math.exp(-1 * payment_chance_modificator * overprice_size)
  local agree_to_pay = math.random() <= payment_chance
  
  if is_over_priced and not agree_to_pay then return false end
```
How simple it all is!

PR: https://github.com/CorsixTH/CorsixTH/pull/3199

--------------

I also attach saves from the original game, with different overpayment percentages, with which you can, firstly, double-check my results, and secondly, verify the inconsistency of the current algorithm for compliance with the behavior of the original game. You just need to load the appropriate save with the desired percentage, buy 2 reception tables and that's it. Rename save before load.

[OverchargeTestSaves.zip](https://github.com/user-attachments/files/24690698/OverchargeTestSaves.zip)
